composer.json

```diff
@@ -48,21 +58,25 @@
     "extra": {
         "tao-extension-name": "taoItems",
         "npm-bridge": {
-            "arguments": ["--prefix", "views"]
+            "arguments": [
+                "--prefix",
+                "views"
+            ]
         }
     },
     "require": {
         "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
-        "oat-sa/generis" : ">=14.0.0",
-        "oat-sa/tao-core" : ">=47.0.0",
-        "oat-sa/extension-tao-backoffice" : ">=6.0.0"
+        "oat-sa/generis": ">=14.0.0",
+        "oat-sa/tao-core": ">=47.0.0",
+        "oat-sa/extension-tao-backoffice": ">=6.0.0",
+        "oat-sa/extension-tao-dac-simple": ">=7.2.0"
```

Comment:
Is that is something we are really need to do? We are leaving no chance to use taoItems without DAC enabled

Reply:
For this migration we need to use class from DAC extension: https://github.com/oat-sa/extension-tao-item/pull/466/files#diff-8ce0e8cb0506f5fae365aa456ba219164e6e2dd6ea217f30c7fe42bd4634caaeR45

I can move this migration to DAC extension, but this new roles will not work properly without READ and/or WRITE permissions (which are provided by DAC).